tools/CMakeLists.txt

```diff
@@ -15,3 +15,40 @@ target_link_libraries( kwiver
                  kwiversys
```

Comment:
For the `kwiver` executable here, I get a cmake error about a duplicate target. I suggest renaming to `kwiver_tool`